### PR TITLE
testAdd tests for stepper.controls_builder.0.dart

### PR DESCRIPTION
**Pull request by ValentinVignal**
Contributes to https://github.com/flutter/flutter/issues/130459

It adds a test for
- `examples/api/lib/material/stepper/stepper.controls_builder.0.dart`

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Data Driven Fixes]: https://github.com/flutter/flutter/wiki/Data-driven-Fixes

### CODE REVIEW

**Review by bleroux — APPROVED** [review]
LGTM!

File: examples/api/test/material/stepper/stepper.controls_builder.0_test.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/stepper/stepper.controls_builder.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('It should override the control builder to display custom buttons', (WidgetTester tester) async {
```
Comment (TahaTesser) [inline examples/api/test/material/stepper/stepper.controls_builder.0_test.dart:10]: What is "It"? Please describe the subject

**Review by TahaTesser — CHANGES_REQUESTED** [review]

File: examples/api/test/material/stepper/stepper.controls_builder.0_test.dart (hunk 1 above)
Comment (ValentinVignal) [reply examples/api/test/material/stepper/stepper.controls_builder.0_test.dart]: I changed the test description in [test: Change test description](https://github.com/flutter/flutter/pull/150669/commits/6da71d3cb8c72bfad6b504554ad55ad8518662a7). Is that an acceptable description?

File: examples/api/test/material/stepper/stepper.controls_builder.0_test.dart [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/stepper/stepper.controls_builder.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('The control builder can be overridden to display custom buttons', (WidgetTester tester) async {
```
Comment (TahaTesser) [inline examples/api/test/material/stepper/stepper.controls_builder.0_test.dart:10]:
```suggestion
  testWidgets('Stepper control builder can be overridden to display custom buttons', (WidgetTester tester) async {
```

**Review by TahaTesser — APPROVED** [review]
LGTM with minor fix

**Pull request merged**